content/go/concepts/comments/comments.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Comments'
+Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code. Single-line comments are created using two consecutive forward slashes. The compiler ignores any text after // on the same line. go // This line will denote a comment in Go  Multi-line comments are created using /* to begin the comment, and */ to end the comment. The compiler ignores any text in between. go'
```

Comment:
```suggestion
Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.'
```